internal/batches/executor/execution_cache.go

```diff
@@ -157,16 +157,16 @@ type JSONCacheWriter interface {
 	WriteAfterStepResult(key string, value execution.AfterStepResult)
 }
 
-type JSONLinesCache struct {
+type SSBCCache struct {
```

Comment:
Plz no acronyms!

```suggestion
type ServerSideCache struct {
```

Reply:
Says the guy who named it _JSON_ lines cache before 😆

Reply:
`JavaScriptObjectNotationLines` killed the linter